includes/classes/class-plugin-manager.php

```diff
@@ -0,0 +1,409 @@
+<?php
+/**
+ * Class PerformanceLab\Plugin_Manager
+ *
+ * @package PerformanceLab
+ * Author: WordPress Performance Team
+ * Author URI: https://make.wordpress.org/performance/
+ * License: GPLv2 or later
+ * License URI: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ */
+
+namespace PerformanceLab;
+
+use WP_Error;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * General purpose class for managing the UI of standalone performance plugins
+ * and rendering of the UI associated therewith.
+ *
+ * @since n.e.x.t
+ * @access public
+ * @ignore
+ */
+class Plugin_Manager {
```

Comment:
Looking at this class, I don't think there's enough justification for it, given the rest of this plugin is coded with functions:
* There are only 3 methods in total.
* 2 of them are public, and 1 of them wouldn't be harmful to be public as well (even though it doesn't need to).
* The methods are static, so there's no testability benefit either.

While generally I'm all for classes, including this class here in the way it is IMO doesn't bring a notable benefit. Its methods are static, there is only a single private method, and the class is only used in one place.

I would suggest you move these three methods as regular functions into an `admin/plugins.php` file, and prefix their name with `perflab_` similar to the other functions. Alternatively, try to use a more object oriented approach with the class, where you avoid static methods and e.g. the code in `admin/load.php` instantiates this class and then calls the method.

Reply:
@felixarntz this has been addressed and I've refactored it all as standalone functions, thank you. Addressed in the following commits:

- https://github.com/WordPress/performance/pull/864/commits/9fd2536edae33cb7ef1010500a9c52cbb6b6e790
- https://github.com/WordPress/performance/pull/864/commits/6248c3b61c26fc7709cd6efb286d6b224cde49ab
- https://github.com/WordPress/performance/pull/864/commits/b3e2236eaf2be687c10378b52008f3e05f1ef5be
- https://github.com/WordPress/performance/pull/864/commits/4b83a796e1b0af819c235669b78ef504757fe0f3
- https://github.com/WordPress/performance/pull/864/commits/822212d9c18ec96c75e8357360e1a319967264e7